### ISSUE TITLE
Activity

### ISSUE THREAD

**Issue by Zweistein2** [code-disaster/steamworks4j#99]
Hey, it seems to me that this repo is more or less dead. Pull requests are no longer merged, issues are not answered and there have been no commits for 10 months now. Is this true? Is it really dead?

**Comment by AlrikG** [comment]
It is the repo of code-disaster. He is the only one who is able to merge PRs. You can fork this repo, but the main problem is to compile the libs for the different platforms. Maybe there is a docker image which allows to cross compile the libs.

**Comment by code-disaster** [comment]
No, it's not dead. Just very low priority to me personally for about a year now. Most missing and requested features (like Input, Inventory, RemotePlay) are relatively complex to add _and test_ in isolation, which doesn't help motivation either.

**Comment by code-disaster** [comment]
> You can fork this repo, but the main problem is to compile the libs for the different platforms. Maybe there is a docker image which allows to cross compile the libs.

Yeah, continuously recompiling and testing the native libraries on all three platforms adds some mental friction, too. I want to have a look into GitHub Actions for that. If someone is willing to contribute at that front, feel free to do so.
